pandas/core/series.py

```diff
@@ -4494,6 +4498,15 @@ def apply(
         Helsinki    2.484907
         dtype: float64
         """
+        if convert_dtype is lib.no_default:
+            convert_dtype = True
+        else:
+            warnings.warn(
+                "the convert_dtype parameter is deprecated and will be removed in a "
+                "future version.",
```

Comment:
Would be good to mention the alternative here if convert_dtype=False was specified